scripts/flattener.py

```diff
@@ -961,16 +966,14 @@ def main(mfinal_id):
 	# For ATAC datasets, assumption is that there is no scale.data, and raw count is taken from mfinal_adata.raw.X
 	raw_matrix_mapping = []
 	cell_mapping_rev_dct = {}
-	normalize_drop = []
-	redundant = []
 	if summary_assay == 'RNA':
-		# If raw matrices are annotated to multiple gencode versions, concatenate on ensembl ID
+		# If raw matrices are annotated to multiple gencode versions, concatenate on ensembl ID and remove ambiguous symbols
 		if len(mfinal_obj.get('genome_annotations',[])) > 1:
-			reconcile_results = reconcile_genes(mfinal_obj, cxg_adata_lst, mfinal_adata.var.index.to_list())
+			reconcile_results = reconcile_genes(cxg_adata_lst)
 			cxg_adata_raw = reconcile_results[0]
-			collapse = reconcile_results[1]
-			redundant = reconcile_results[2]
-			collapse_dict = reconcile_results[3]
+			redundant = reconcile_results[1]
+			all_remove = reconcile_results[2]
```

Comment:
combine these lines to `cxg_adata_raw, redundant, all_remove = reconcile_genes(cxg_adata_lst)`?

Reply:
Ooh, nice, much better. I'll give it a try.